core/domain/collection_services.py

```diff
@@ -491,6 +491,36 @@ def get_collection_summaries_matching_query(query_string, cursor=None):
     ], search_cursor)
 
 
+def get_displayable_collections_matching_query(query_string,
```

Comment:
Changed. However I couldn't move this function to `core.domain.summary_services`